includes/options/class-amp-options-menu.php

```diff
@@ -272,45 +275,148 @@ public function render_validation_handling() {
 	}
 
 	/**
-	 * Post types support section renderer.
+	 * Supported templates section renderer.
 	 *
-	 * @since 0.6
+	 * @since 1.0
 	 */
-	public function render_post_types_support() {
-		$builtin_support = AMP_Post_Type_Support::get_builtin_supported_post_types();
-		$element_name    = AMP_Options_Manager::OPTION_NAME . '[supported_post_types][]';
+	public function render_supported_templates() {
+
+		// @todo If AMP theme support is native and required, let all_templates_supported be true? Maybe template support could have an all_templates_supported flag?
 		?>
+		<fieldset id="all_templates_supported_fieldset">
+			<p>
+				<label for="all_templates_supported">
+					<input id="all_templates_supported" type="checkbox" name="<?php echo esc_attr( AMP_Options_Manager::OPTION_NAME . '[all_templates_supported]' ); ?>" <?php checked( AMP_Options_Manager::get_option( 'all_templates_supported' ) ); ?>>
+					<?php esc_html_e( 'Serve all templates as AMP regardless of what is being queried.', 'amp' ); ?>
+				</label>
+			</p>
+			<p class="description">
+				<?php esc_html_e( 'This will allow all of the URLs on your site to be served as AMP by default.', 'amp' ); ?>
+			</p>
+		</fieldset>
+
+		<fieldset id="singular_templates_fieldset">
+			<?php $element_name = AMP_Options_Manager::OPTION_NAME . '[supported_post_types][]'; ?>
+			<h4 class="title"><?php esc_html_e( 'Content Types', 'amp' ); ?></h4>
+			<p>
+				<?php esc_html_e( 'The following content types will be available as AMP by default, but you can override this on an item-by-item basis:', 'amp' ); ?>
```

Comment:
Maybe this would be enough:
>The following content types will be available as AMP:

If they're checked by default, that might communicate "will be available as AMP **by default**"

<img width="1389" alt="content-types" src="https://user-images.githubusercontent.com/4063887/42111298-90b8ce7e-7ba9-11e8-856c-cfaeb316c7ff.png">

Reply:
Actually they would be available as AMP by default, no?

Reply:
Yes, they would be available by default. But I was thinking that it might not be necessary to mention that. 

But on second thought, that description is probably helpful. Most people probably don't want to disable AMP on posts, for example.